base_location_nuts/views/res_partner_view.xml

```diff
@@ -1,55 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
-<data>
+<odoo>
 
 <record model="ir.ui.view" id="view_partner_form_nuts">
     <field name="name">Partner form with NUTS</field>
     <field name="model">res.partner</field>
     <field name="inherit_id" ref="base.view_partner_form"/>
     <field name="arch" type="xml">
         <xpath expr="//sheet/group//field[@name='country_id']/.." position="after">
-            <div width="1%" cell-class="oe_form_group_cell_label"
-                 attrs="{'invisible': [('use_parent_address','=',True)]}">
-                <field name="lbl_substate" class="oe_inline"/>
-            </div>
-            <div attrs="{'invisible': [('use_parent_address','=',True)]}">
-                <field name="substate"
-                       class="oe_no_button"
-                       options="{'no_open': True, 'no_create': True}"
-                       attrs="{'readonly': [('use_parent_address','=',True)]}"/>
-            </div>
-            <div width="1%" cell-class="oe_form_group_cell_label"
-                 attrs="{'invisible': [('use_parent_address','=',True)]}">
-                <field name="lbl_region" class="oe_inline"/>
-            </div>
-            <div attrs="{'invisible': [('use_parent_address','=',True)]}">
-                <field name="region"
-                       class="oe_no_button"
-                       options="{'no_open': True, 'no_create': True}"
-                       attrs="{'readonly': [('use_parent_address','=',True)]}"/>
-            </div>
+            <field name="nuts1_id"/>
```

Comment:
I didn't. That change comes from a previous commit (https://github.com/Tecnativa/partner-contact/commit/eade50cd3ab61b354346e8bb6e227508d59008c3)